src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java

```diff
@@ -177,4 +184,10 @@ public void addErrorListenerScript() {
             + "function (e, u, l, c, errorObj) { window.errors = errorObj.stack }';"
             + "document.querySelector('body').appendChild(script);");
   }
+
+
+  public Long getCurrentPosition() {
+    Long value = (Long) js.executeScript("return window.pageYOffset;");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "value". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1488)